XamlControlsGallery/ControlPages/AcrylicPage.xaml.cs

```diff
@@ -27,7 +27,7 @@ private void Slider_ValueChanged(object sender, RangeBaseValueChangedEventArgs e
         {
             //Rectangle shape = (sender == OpacitySliderInApp) ? CustomAcrylicShapeInApp : CustomAcrylicShape;
             Rectangle shape = CustomAcrylicShapeInApp;
-            if (sender == OpacitySliderLumin)
+            if ((Slider)sender == OpacitySliderLumin)
```

Comment:
Why is this change necessary? I would expect comparing 'object' to 'Slider' should work without the need to down cast.